app/views/hyrax/batch_edits/_delete_selected.html.erb

```diff
@@ -3,6 +3,6 @@
   <%= form_tag(hyrax.batch_edits_path, method: :delete, class: "batch-select-all hidden button_to-inline", "data-behavior" => 'batch-select-all') do -%>
       <%= hidden_field_tag('update_type', 'delete_all') %>
       <%= hidden_field_tag('return_controller', params[:controller]) %>
-    <%= submit_tag("Delete Selected", class: 'batch-all-button btn btn-danger submits-batches', data: { confirm: "Deleting a file from #{application_name} is permanent. Click OK to delete this file from #{application_name}, or Cancel to cancel this operation" }) %>
+    <%= submit_tag(t("hyrax.batch_edits.delete_selected.button_label"), class: 'batch-all-button btn btn-danger submits-batches', data: { confirm: "Deleting a file from #{application_name} is permanent. Click OK to delete this file from #{application_name}, or Cancel to cancel this operation" }) %>
```

Comment:
Can we break this one long line into multiple shorter ones to make it easier to read?